openedx/openedx-platform
# feat: Adds discussions settings for new discusions experience [BD-38] [TNL-8621] [BB-4854]

## Description

This commit adds new discussions settings for the new discussions experience. These are stored in the course so they can be a part of course import/export flow.
These are also added to the discussions configuration API to allow MFEs to update the settings.

## Supporting information

- [TNL-8621](https://openedx.atlassian.net/browse/TNL-8621)

## Testing instructions

1.  Use the updates course authoring MFE from here: https://github.com/edx/frontend-app-course-authoring/pull/197
2.  Change the legacy provider settings for anonymous posting settings, and make sure they are persisted
3.  The settings should be saved to the coruse structure as well which you can verify using the shell

## Deadline

"None"
